nubison_model/Model.py

```diff
@@ -22,6 +21,53 @@ def load_model(self) -> None: ...
     def infer(self, input: Any) -> Any: ...
 
 
+class NubisonMLFlowModel(PythonModel):
```

Comment:
In this class, other than modularizing the symlink functionality, is there any other intention behind the changes?